packages/auth/src/accessToken/service.test.ts

```diff
@@ -251,29 +248,31 @@ describe('Access Token Service', (): void => {
         token.managementId,
         token.value
       )
-      expect(result.success).toBe(true)
-      expect(result.success && result.value).not.toBe(originalTokenValue)
+      assert.ok(result)
+      expect(result.value).not.toBe(originalTokenValue)
```

Comment:
```suggestion
      expect(result.value).not.toBe(originalTokenValue)
      await expect(accessTokenService.get(token)).resolves.toBeUndefined()
```
Not sure if it's worth also checking `getByManagementId`, `introspect`, `revoke`, or `rotate`

Reply:
Do you mean checking whether the old one can be fetched by `getByManagementId`/`introspect`/`revoke`/`rotate` ?

Reply:
correct

Reply:
Alternatively, each of those look up functions could have a `cannot _____  rotated token` test